tests/Doctrine/Tests/ORM/Functional/ResultCacheTest.php

```diff
@@ -168,6 +169,21 @@ public function testEnableResultCache(): void
         $this->_em->getConfiguration()->setResultCacheImpl(new ArrayCache());
     }
 
+    public function testEnableResultCacheWithIterable(): void
+    {
+        $cache = new ArrayCache();
+        $query = $this->_em->createQuery('select ux from Doctrine\Tests\Models\CMS\CmsUser ux');
+
+        $query->enableResultCache();
+        $query->setResultCacheDriver($cache);
+        $query->setResultCacheId('testing_iterable_result_cache_id');
+        iterator_to_array($query->toIterable());
+
+        $this->assertTrue($cache->contains('testing_iterable_result_cache_id'));
+
+        $this->_em->getConfiguration()->setResultCacheImpl(new ArrayCache());
```

Comment:
:thinking: I don't understand why that last line is not an assertion… what's the point of doing something if you don't perform assertions afterwards, or setup expectations beforehand?

Reply:
Oh ok, it makes sense then 👍